hack/relprep.sh

```
@@ -157,7 +148,11 @@ echo "Updating the kedacontrollers crd from code and copying it to $ver manifest
 make manifests
 cp config/crd/bases/keda.sh_kedacontrollers.yaml keda/${ver}/manifests/
 # revert any changes to the kustomization
-git co config/manager/kustomization.yaml
+git checkout config/manager/kustomization.yaml config/default/kustomization.yaml
+
+echo "Syncing bundle annotations to keda/${ver}/metadata/ (stripping test-only entries)"
+sed '/operators\.operatorframework\.io\.test\./d; /^[[:space:]]*#/d; /^[[:space:]]*$/d' \
+    bundle/metadata/annotations.yaml > keda/${ver}/metadata/annotations.yaml
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)